### PR TITLE
Ensure portlet configuration for default content

### LINKED ISSUE #1214: Portlet assignments for Inbox, Templates and Contacts are missing on new setup
On a newly set up GEVER from `4.5-stable`, but probably also `master`, the portlet assignments for the `Inbox`, `Templates` and `Contacts` global tabs are missing.

Once fixed this should probably be backported to `4.5-stable`.

### PR DESCRIPTION
**Pull request by phgross**
Currently the portlet configuration of default content is done in profilehooks. Wich has been copied all over the policies. 

This PR fix this situation, by adding EventHandlers for content with a special portlet configuration. So it doesn't matter if the content is created by og.setup, OGGBundle or the API the portlet configuration is always as expected. Closes #1214

Short overview:
**Personal overview**: No context assignments, Inheritance active
**Inbox**: Navigation portlet, Inheritance blocked
**InboxContainer**: Navigation portlet, Inheritance blocked
**TemplateFolder**: Navigation portlet, Inheritance blocked
**Sub-TemplateFolder**: No context assignments, Inheritance active
**Contactfolder**: No context assignments, Inheritance blocked
**Sitzungen**: No context assignments, Inheritance blocked

### CODE REVIEW

File: opengever/inbox/subscribers.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from opengever.base.portlets import block_context_portlet_inheritance
+from opengever.base.portlets import add_navigation_portlet_assignment
+
+
+def configure_inboxcontainer_portlets(container, event):
+    """Block portlet inheritance.
```
Comment (lukasgraf) [inline opengever/inbox/subscribers.py]: Somehow this configures the inbox navigation portlet differently than we usually do: `[x] Zeige nur den Inhalt des aktuellen Ordners.` is enabled, and therefore leads to weird effects (the navigation portlet disappearing again if you visit a sub-object of the inbox).

File: opengever/inbox/subscribers.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    """
+
+    block_context_portlet_inheritance(container)
+
+
+def configure_inbox_portlets(inbox, event):
+    """Block portlet inheritance and add navigation portlet.
+    """
+    block_context_portlet_inheritance(inbox)
+    add_navigation_portlet_assignment(inbox, currentFolderOnly=True)
```
Comment (lukasgraf) [inline opengever/inbox/subscribers.py]: Why `currentFolderOnly=True`? This configures the inbox navigation portlet differently than we usually do: `[x] Zeige nur den Inhalt des aktuellen Ordners.` is enabled, and therefore leads to weird effects (the navigation portlet disappearing again if you visit a sub-object of the inbox).

File: opengever/policytemplates/policy_template/opengever.+package.name+/opengever/+package.name+/hooks.py.bob [hunk 3]
```diff
@@ -1,19 +1,7 @@
-from opengever.setup.hooks import assign_default_navigation_portlet
```
Comment (phgross) [inline opengever/policytemplates/policy_template/opengever.+package.name+/opengever/+package.name+/hooks.py.bob]: Yes!

File: opengever/setup/hooks.py [hunk 4]
```diff
@@ -44,36 +37,3 @@ def assign_roles(context, admin_groups):
...
-            topLevel=1,
-            bottomLevel=0)
-
-    # Block inherited context portlets on content
-    assignable = getMultiAdapter(
-        (content, manager), ILocalPortletAssignmentManager)
-    assignable.setBlacklistStatus(CONTEXT_CATEGORY, True)
-
-
-def block_context_portlets(site, content_id):
```
Comment (phgross) [inline opengever/setup/hooks.py]: Yes, but that's the plan in may opinion.

**Comment by phgross** [comment]
@lukasgraf @deiferni updated ...

**Pull request merged**